lib/cgrates_web_jsonapi_web/views/cdr_stat_view.ex

```diff
@@ -13,4 +13,8 @@ defmodule CgratesWebJsonapiWeb.CdrStatView do
     :total_normal_clearing_disconnects,
     :total_rejected_disconnects
   ])
+
+  def render("extra_fields.json", %{data: data}) do
+    %{data: %{columns: data.columns, num_rows: data.num_rows, rows: data.rows}}
```

Comment:
It's not clear what these (columns, num_rows and row) fields mean. I expected this endpoint will return JSONAPI response like:

```
{
  "data": [
     {
        "type": "cdr-extra-fields",
        "id": <field-name>
     }
  ]
}
```

Or a simple array of strings

Reply:
We also need swagger documentation